master/buildbot/schedulers/forcesched.py

```diff
@@ -121,7 +123,9 @@ def getFromKwargs(self, kwargs):
 
     def updateFromKwargs(self, properties, kwargs, **unused):
         """Primary entry point to turn 'kwargs' into 'properties'"""
-        properties[self.name] = self.getFromKwargs(kwargs)
+        value = self.getFromKwargs(kwargs)
+        if value is not None:
```

Comment:
It will be much more readable when you write inverse condition. I mean, exception (value is None) has own branch, but rest of code is executed in the main branch (flat).

So, my suggestion is that, rewrite it in this way:
```
if value is None:
    return
 properties[self.name] = value
```

Reply:
Sure